src/components/App.js

```diff
@@ -23,11 +24,44 @@ class App extends React.Component {
     return (
       <Provider store={store}>
         <I18nextProvider i18n={i18n}>
-          <Router
-            history={browserHistory}
-            children={routes}
-            render={applyRouterMiddleware(useScroll())}
-          />
+          <div>
+            <Helmet defaultTitle='inspire.data.gouv.fr'>
+              <meta charSet='utf-8' />
+              <meta name='viewport' content='width=device-width, initial-scale=1' />
+
+              <meta name='twitter:card' content='summary' />
+              <meta name='twitter:site' content='@geodatagouv' />
+              <meta name='twitter:title' content='Passerelle Inspire' />
+              <meta name='twitter:description' content='Trouvez facilement les données géographiques dont vous avez besoin' />
+              <meta name='twitter:image' content='http://inspire.data.gouv.fr/assets/inspire-logo.jpg' />
+
+              <meta property='og:title' content='Passerelle Inspire' />
+              <meta property='og:type' content='website' />
+              <meta property='og:url' content='http://inspire.data.gouv.fr/' />
+              <meta property='og:image' content='http://inspire.data.gouv.fr/assets/inspire-logo.jpg' />
+              <meta property='og:description' content='Trouvez facilement les données géographiques dont vous avez besoin' />
+              <meta property='og:site_name' content='Passerelle Inspire' />
+
+              <meta name='theme-color' content='#ffffff' />
+              <link rel='apple-touch-icon' sizes='180x180' href='/favicons/apple-touch-icon.png' />
+              <link rel='icon' type='image/png' href='/favicons/favicon-32x32.png' sizes='32x32' />
+              <link rel='icon' type='image/png' href='/favicons/favicon-16x16.png' sizes='16x16' />
+              <link rel='manifest' href='/favicons/manifest.json' />
+              <link rel='mask-icon' href='/favicons/safari-pinned-tab.svg' color='#5bbad5' />
+
+              <link rel='stylesheet' href='https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.2.10/components/reset.min.css' />
```

Comment:
all these things that don’t change should stay in index.html, we should only keep the things that can change when navigating to a new route in here